.pylintrc

```diff
@@ -0,0 +1,23 @@
+[MASTER]
+jobs=0
+ignore-paths=
+    cli/tests
```

Comment:
This seems to also be leftover from snafu, could remove this ignore path.

Reply:
same thing as the problem with the bandit exclusions. The cli testsuses fixtures which trigger `unused-argument` or `unsued-import` if the fixture is ffrom a different module. Instead of littering all the tests with pylint disables I just excluded the tests from linting since I didn't have much luck catching many issues there with them